src/main/java/org/broadinstitute/hellbender/tools/spark/sv/AlignAssembledContigsSpark.java

```diff
@@ -84,4 +76,24 @@ protected void runTool(final JavaSparkContext ctx) {
         assembledBreakpoints.saveAsTextFile(output);
     }
 
+    static JavaPairRDD<String, ContigsCollection> getContigsCollectionKeyedByBreakpointId(final JavaSparkContext ctx, final String inputPath) {
+        final JavaRDD<String> inputAssemblies = ctx.textFile(inputPath).cache();
+
+        final JavaPairRDD<String, String> contigCollectionByBreakpointId =
+                inputAssemblies
+                        .flatMapToPair(RunSGAViaProcessBuilderOnSpark::splitAssemblyLine);
+
+        return contigCollectionByBreakpointId.mapValues(ContigsCollection::fromPackedFasta);
+    }
+
+    /**
+     * input format is the text representation of an alignment region
+     * @param alignedAssemblyContigLine An input line with the tab-separated fields of an alignment region
+     * @return A tuple with the breakpoint ID and string representation of an AssembledBreakpoint, or an empty iterator if the line did not have two comma-separated values
+     */
+    static AlignmentRegion parseAlignedAssembledContigLine(final String alignedAssemblyContigLine) {
+        final String[] split = alignedAssemblyContigLine.split("\t", -1);
```

Comment:
Method doesn't seem to be used in this tool but rather `CallVariantsFromAlignedContigsSpark`. Would it make sense to move it there?